src/main/scala/net/akmorrow13/endive/EndiveConf.scala

```diff
@@ -91,6 +91,11 @@ class EndiveConf extends Serializable {
 
   @BeanProperty var sample: Boolean = true
 
+  /* Configuration values used for saving test set */
+  @BeanProperty var tf: String = null
+  @BeanProperty var cellTypes: String = null
+  @BeanProperty var hasSequences: Boolean = true
```

Comment:
This is for dataset creation pipeline. If sequences are saved that step is skipped